gutenberg.php

```diff
@@ -187,6 +189,16 @@ function gutenberg_init( $return, $post ) {
 	add_filter( 'screen_options_show_screen', '__return_false' );
 	add_filter( 'admin_body_class', 'gutenberg_add_admin_body_class' );
 
+	$post_type_object = get_post_type_object( $post_type );
+	/**
```

Comment:
Should be `/*`, not `/**`

>Important note: Multi-line comments must not begin with /** (double asterisk) as the parser might mistake it for a DocBlock. Use /* (single asterisk) instead.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-2-multi-line-comments

Reply:
Updated in 2667bf463d5a1156f0e4833be55516b8cc208431